src/bot.py

```diff
@@ -77,6 +79,17 @@ def draw_graph(self):
         self.machine = GraphMachine(model=self, states=self._states, initial="hero_selection", transitions=self._transitions, queued=True)
         self.machine.get_graph().draw('my_state_diagram.png', prog='dot')
 
+    def sort_belt(self):
+        # here instead because pickit depends on belt_manager and belt_manager depends on pickit
+        wait(0.4,0.6)
+        keyboard.send(self._config.char["show_belt"]) #toggle belt
+        wait(0.1,0.2)
+        img = self._screen.grab()
+        belt_contents, self._belt_manager.belt_height = self._belt_manager.get_belt_contents(img,1,0)
+        self._belt_manager.drop_wrong_belt_pots(belt_contents,0) #drops wrong potions and generates pots_remaining
+        _ = self._pickit.pick_up_items(self._char) #pickup compatible pots
+        keyboard.send(self._config.char["show_belt"]) #toggle belt
+
```

Comment:
This should be part of the belt manager. Why does BeltManager depend on Pickit? I don't really get the pickit call after drop wrong belt pots. Do you drop them and then pick up again to make sure they are in the right order?